src/work-allocation/containers/available-tasks/available-tasks.component.ts

```diff
@@ -50,15 +50,16 @@ export class AvailableTasksComponent extends TaskListWrapperComponent {
   }
 
   public loadTasks(): void {
-    if (this.selectedLocations) {
-      super.loadTasks();
-    }
+    // TODO: Need to check whether this functionality is needed
+    // if (this.selectedLocations) {
+    super.loadTasks();
```

Comment:
Left TODO in there because unsure whether the use of the overload was necessary. Considering everything works (apart from the location filter which I don't believe is related to this) we can probably just call loadTasks()

Reply:
Removed this method